src/main/java/name/webdizz/sonar/grammar/spellcheck/GrammarChecker.java

```diff
@@ -15,10 +15,12 @@
     public static final String DEFAULT_DICT_PATH = "src/main/resources/dict/english.0";
     private static final Logger LOGGER = LoggerFactory.getLogger(GrammarChecker.class);
     private SpellDictionary dictionary;
-    private final GrammarDictionaryLoader dictionaryLoader;
+    private GrammarDictionaryLoader dictionaryLoader;
+    private int minimumWordLengths;
 
-    public GrammarChecker(final GrammarDictionaryLoader dictionaryLoader) {
+    public GrammarChecker(final GrammarDictionaryLoader dictionaryLoader, int minimumWordLengths) {

```

Comment:
HI Izzet,
Could you please clarify how to use configuration, I noticed that somehow it is possible to use IoC, but now I'm not really sure how. For now the only solution I see it is to pass Setting settings instead of passing the minimumWordLengths and then get minimumWordLengths  like this (settings.getString(GrammarPlugin.MIN_WORD_LENGTH)